homeassistant/components/binary_sensor/ping.py

```diff
@@ -35,6 +35,9 @@
 PING_MATCHER = re.compile(
     r'(?P<min>\d+.\d+)\/(?P<avg>\d+.\d+)\/(?P<max>\d+.\d+)\/(?P<mdev>\d+.\d+)')
 
+PING_MATCHER_ALPINE = re.compile(
```

Comment:
We should rename `PING_MATCHER_ALPINE` to  `PING_MATCHER_BUSYBOX`. I guess that the same issue will appear if people start to use Home Assistant on OpenWRT.